### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by TakeruKondo**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1488"></a>

### CODE REVIEW

File: public/data/takerukondo.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "avatar": "https://github.com/TakeruKondo.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TakeruKondo",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/TakeruKondo",
```
Comment (kumarsonsoff3) [inline public/data/takerukondo.json:14]: You have added a link to a _**Suspended Account**_! Please check if you've entered the correct Twitter username in the link, else, update the link with a new one! 

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/takerukondo.json (hunk 1 above)
Comment (EmmaDawsonDev) [reply public/data/takerukondo.json:14]: @TakeruKondo please update or remove the Twitter link

**Comment by EmmaDawsonDev** [comment]
Do you need any help with this @TakeruKondo

**Comment by EmmaDawsonDev** [comment]
Since this is more than a month old I’m going to close it for now. Please feel free to make a new pull request when you’ve had time to make the changes.

**Pull request closed without merging**